pootle/static/css/scores.css

```diff
@@ -139,7 +138,14 @@ html[dir="rtl"] .path-summary-more .top-scorers-table tr td.number:last-child
 .path-summary-more .top-scorers-table tr td.number:first-child
 {
     text-align: left;
-    width: 2em;
+    width: 1em;
+    padding-right: 1em;
+}
+
+html[dir="rtl"] .path-summary-more .top-scorers-table tr td.number:first-child {
```

Comment:
I know this is not strictly part of this PR, but I realize now and wanted to flag that this CSS selector is unnecessarily complicated.

Also, nit: braces go on their own line.